bbb-core-api/cmd/api/helpers.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"crypto/sha1"
+	"crypto/sha256"
+	"crypto/sha512"
```

Comment:
We support 384 too, if this is used where I think it's used: https://github.com/bigbluebutton/bigbluebutton/blob/v2.7.x-release/bigbluebutton-web/grails-app/conf/bigbluebutton.properties#L328